app/models/node.rb

```diff
@@ -20,6 +20,8 @@ class Node < ApplicationRecord
   enumerize :state, in: STATES, scope: true
   content_attribute :content_body
   content_attribute :name
+  # options is not currently versioned but could be in the future
+  store_attribute :content, :options
```

Comment:
Feels a bit strange to have options as part of content.... couldn't we put this in 'data'? (The other jsonb field, which NewsArticle uses to store its release date...)